### PR TITLE
Updated to current build process

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -3,40 +3,146 @@ workspace:
...
     commands:
-      - go build -v -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/amd64/drone-git
+      - |
+        if test "${DRONE_TAG}" = ""; then
+          go build -v -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/amd64/drone-git
+        else
+          go build -v -ldflags "-X main.version=${DRONE_TAG##v} -X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/amd64/drone-git
+        fi
+
+  # build_linux_i386:
```
Comment (tonglil) [inline .drone.yml]: Commented out?

Comment (tboerger) [reply .drone.yml]: This also requires a native agent because system packages are involved. We can cross build only go binary images.

**Comment by tboerger** [comment]
@bradrydzewski maybe you want to give it a final review to get it merged like the other build process updates?

**Comment by bradrydzewski** [comment]
I would suggest we archive this repository and not make any big changes. In 0.9 it is replaced by drone-git, and any changes we make to this repository are high risk, since they could impact active Drone installs.

**Comment by tboerger** [comment]
I guess we won't leave it directly because not everybody could directly switch to 0.9 because of the massive changes.

**Comment by tboerger** [comment]
After chatting with brad about that I'm merging this PR and directly test the functionality of the plugin.

**Pull request merged**